backend/api/Database/Models/VideoStream.cs

```diff
@@ -23,5 +23,8 @@ public class VideoStream
         [MaxLength(64)]
         [Required]
         public string Type { get; set; }
+
+        [MaxLength(64)]
+        public bool ShouldRotate { get; set; }
```

Comment:
Should we specify the degree to rotate here?

Reply:
Should it be required?

Reply:
We could, but currently we only support 270 deg rotation. Would renaming to ShouldRotate270 be better?

Reply:
I don't think it should be required as this is only relevant for some robots

Reply:
Agreed, should also state CW or CCW